browser/extensions/api/brave_rewards_api.cc

```diff
@@ -431,49 +417,95 @@ ExtensionFunction::ResponseAction BraveRewardsGetCurrentReportFunction::Run() {
   return RespondNow(NoArguments());
 }
 
-BraveRewardsGetGrantsFunction::~BraveRewardsGetGrantsFunction() {
+BraveRewardsFetchPromotionsFunction::~BraveRewardsFetchPromotionsFunction() {
```

Comment:
Destructor should be defaulted if no specific logic required; 

`BraveRewardsFetchPromotionsFunction::~BraveRewardsFetchPromotionsFunction() = default;`